.gitmodules

```diff
@@ -8,6 +8,7 @@
 [submodule "Lib/LVGL9/lvgl"]
 	path = Lib/LVGL9/lvgl
 	url = git@github.com:lvgl/lvgl.git
-[submodule "Drivers/SC24_ILI9488_DRIVER"]
-	path = Drivers/SC24_ILI9488_DRIVER
+
+[submodule "Drivers/ILI9488_DRIVER"]
+	path = Drivers/ILI9488_DRIVER
 	url = https://github.com/giuuuug/ILI9488_DRIVER
```

Comment:
Don't link to personal stuff! The driver either is a SC organization repository or the original one. Hence if you have new stuff to add to the driver, ask someone to fork it in the SC organization from the original repo and start from there.

Reply:
It's ok to link to other people stuff only if the project seems to be big and probably will never be removed from the face of the earth. But even in that case would be best to fork it on the SC organization GitHub profile.